src/mlpack/methods/softmax_regression/softmax_regression.hpp

```diff
@@ -141,13 +140,8 @@ class SoftmaxRegression
    * @param numClasses Number of classes for classification.
    * @return Objective value of the final point.
    */
-  double Train(const arma::mat &data, const arma::vec& labels,
-               const size_t numClasses);
-
-  //! Sets the size of the input vector.
-  size_t& InputSize() { return inputSize; }
-  //! Gets the size of the input vector.
-  size_t InputSize() const { return inputSize; }
```

Comment:
Oops, you were right, it is `n_cols` not `n_rows`.  Sorry for the confusion.  I see you also already wrote `FeatureSize()`, so nothing for me to do. :)